katas/joaquin-gioffre/main/Week-01/killer-garage-door/Door.cs

```diff
@@ -0,0 +1,21 @@
+namespace Week1;
+
+public class Door
+{
+    public int Position = 0;
+    public string State = "closed"; 
+    public const int OPENED_POSITION = 5;
+    public const int CLOSED_POSITION = 0;
+
+    public void ChangeDoorPosition()
+    {
+        if (State == "opening" && Position < OPENED_POSITION)
```

Comment:
You would have to create an enum like this:
```csharp
public enum DoorState
{
  Opening,
  Closing,
  Stopped,
  Open,
  Closed
}
```
And use it like this:
```csharp
public string State { get; set; } = DoorState.Closed;
```
And anywhere you need to compare it to a value it would be just like this:
```csharp
door.State == DoorState.Closed;
```